Resources/Migrations/deltaMigrations.yml

```diff
@@ -146,3 +146,6 @@ BarSignWhiskeyEcho: BarSignWhiskeyEchoes
 # 2025-03-10
 MetempsychoticMachine: null
 MetempsychoticMachineCircuitboard: null
+
+# 2025-03-12
+SmartFridge: DVSmartFridge
```

Comment:
changed to medical as its only mapped in med

Reply:
![Screenshot_20250316-095557](https://github.com/user-attachments/assets/caae8c3a-b41d-43df-a0e9-b531a1e8029e)

Reply:
mapping effort for 1 map vs mapping effort for every other map

Reply:
ok fair enough